crypto/stock-exchange/README.md

```diff
@@ -0,0 +1,120 @@
+# Stock Exchange
+
+## Firestore Collections
+
+* Stocks
+> this will contain details of all stocks that should be listed
+
+## Stock Object
+
+```
+{
+  "name": <stock_name>,
+  "price": <listed_price>,
+  "quantity": <available_quantity>
+}
+```
+
+## Trading Request Object
+
+```
+{
+  "tradeType": "buy/sell",
+  "stockName": <name_of_the_stock>,
+  "quantity": <number_of_quantities>,
+  "listedPrice": <listed_stock_price>,
+  "totalPrice": <total_purchase_price>
+}
+```
+
+## Trading Response Object
+
+```
+{
+  "userBalance": <balance left with user after successful trade>
+}
+```
+
+## **Requests**
+
+|               Route                |    Description    |
+| :--------------------------------: | :---------------: |
+|      [GET /stocks](#get-stocks)      | Returns all stocks to be listed |
+|     [POST /stocks](#post-stocks)     | Creates new stock  |
+|     [PATCH /trade/:username](#patch-tradeusername)     | New trading request  |
```

Comment:
Why do we need the username here? Can we not get the user details from the userId in the JWT?

Also, `/trade/new` seems to make a bit more sense about the order being a new trade